model/event/subscriptions_test.go

```diff
@@ -105,7 +107,8 @@ func (s *subscriptionsSuite) SetupTest() {
 				Type:   EmailSubscriberType,
 				Target: &t4,
 			},
-			Owner: "me",
+			Owner:     "me",
+			OwnerType: OwnerTypePerson,
 		},
 	}
```

Comment:
Can you add another subscription, with OwnerType: OwnerTypeProject?

Reply:
added